tc-client/pom.xml

```diff
@@ -43,6 +43,10 @@
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
I'm not quite sure from where this module was getting its slf4j-api library but, with the other changes, this module fails to compile without this dependency declaration.  (Code in this module has direct use of SLF4J so a direct declaration is not out of place.)